src/main/webapp/go.js

```diff
@@ -21,21 +22,38 @@ const DATA_URL = '/go-data';
 const NAME_URL = '/name-data';
 
 // Div IDs that text or a map should be inserted into.
+const HINT_DISPLAY = 'hint-area';
 const RIDDLE_DISPLAY = 'riddle-area';
+const MAP_DISPLAY = 'map-area';
+const MAP_MSSG_DISPLAY = 'map-message-area';
 
 // Hard-coded messages to be displayed to the user.
 const FINAL_MSSG = 'Congrats, you\'ve finished the hunt!';
 
 // Other constants.
 const INDEX_PARAM = 'new-index';
 const INVISIBLE_CLASS = 'invisible';
+const mapKey = config.MAP_KEY;
 
 // Global variables.
 const puzzleArr = [];
 let destIndex;
 const destArr = [];
+let map;
 
-window.onload = getHunt();
+window.onload = function() {
+  addScriptToHead();
+  getHunt();
+};
+
+/**
+ * Add the Map API key to the head.
+ */
+function addScriptToHead() {
+  const newScript = document.createElement('script');
+  newScript.src = 'https://maps.googleapis.com/maps/api/js?key=' + mapKey;
```

Comment:
probably not much value in creating the mapKey variable... would just use config.MAP_KEY directly here.